resources/knative-eventing/Chart.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
 name: knative-eventing
 version: 0.12.0
 tillerVersion: ">=2.7.2-0"
```

Comment:
I guess tillerVersion is not required anymore, since there is no tiller in helm3

Reply:
Done